utility/build.gradle

```diff
@@ -50,6 +50,9 @@ dependencies {
       'com.caverock:androidsvg-aar:1.4',
       'com.github.bumptech.glide:glide:4.9.0',
       'com.google.dagger:dagger:2.24',
+      'com.google.firebase:firebase-analytics-ktx:17.4.2',
+      'com.google.firebase:firebase-crashlytics:17.0.0',
+      'com.google.firebase:firebase-core:17.4.2',
```

Comment:
Nit: Rearrange them in alphabetical order:
```
      'com.google.firebase:firebase-core:17.4.2',
      'com.google.firebase:firebase-crashlytics:17.0.0',
```

Reply:
Done